src/redteam/providers/crescendo/index.ts

```diff
@@ -391,6 +391,13 @@ class CrescendoProvider implements ApiProvider {
     if (response.error) {
       throw new Error(`Error from redteam provider: ${response.error}`);
```

Comment:
maybe this should return undefined as well, we don't want to fail just want to skip the turn?